2-factor-authentication/test/my-first-askui-test-suite.test.ts

```diff
@@ -1,48 +1,73 @@
 import { aui, auiAndroid } from './helper/jest.setup';
 
 describe('jest with askui', () => {
-  it('should click on text', async () => {
+  it('Login in Github', async () => {
 
     await aui.mouseLeftClick().exec();
     await aui.click().text().withText('Sign in').exec();
     await aui
-      .typeIn('<Username>')
+      .typeIn('<usename>', { isSecret: true, secretMask: '**' })
```

Comment:
Was changing `'<Username>'` to `'<usename>'` on purpose? (especially removing the *r*)

Reply:
it was a mistake